### PR TITLE
docs: add readme for local run with training data set

### PR DESCRIPTION
**Pull request by ni507**

### CODE REVIEW

File: source_codes/gralaf/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Run locally with example training data set
+
+A example training data set is provided: [training data set](./dataset/training_data_20230521-202044.csv)
```
Comment (stefenz) [inline source_codes/gralaf/README.md]: An example

File: source_codes/gralaf/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+This training data set can be used to train the model. In the [config file](./config.yaml#L23) the correct training data set must be provided.
+The correct IP must be specified for the configs `prometheus_url` and `lasm_server_urls`.
+
+With the config `use_archive = false` the "live" mode is started and experiments with the chaos mesh dashboard can be injected. This results in a root cause analysis which is sent to the lasm server.
```
Comment (stefenz) [inline source_codes/gralaf/README.md]: I would rather say "and faults can be injected with chaos mesh"

**Review by stefenz — CHANGES_REQUESTED** [review]

File: source_codes/gralaf/README.md (hunk 1 above)
Comment (ni507) [reply source_codes/gralaf/README.md]: Fixed.

File: source_codes/gralaf/README.md (hunk 2 above)
Comment (ni507) [reply source_codes/gralaf/README.md]: Fixed.

**Review by stefenz — APPROVED** [review]

**Pull request merged**